tauri-app/src/lib/services/time.ts

```diff
@@ -0,0 +1,9 @@
+export const TIME_DELTA_24_HOURS = 60 * 60 * 24;
+export const TIME_DELTA_48_HOURS = TIME_DELTA_24_HOURS * 2;
+export const TIME_DELTA_7_DAYS = TIME_DELTA_24_HOURS * 7;
+export const TIME_DELTA_30_DAYS = TIME_DELTA_24_HOURS * 30;
+export const TIME_DELTA_1_YEAR = TIME_DELTA_24_HOURS * 365;
+
+export function nowTimestamp(): number {
```

Comment:
usually with time, because it's non deterministic, we'd have a function that handles times, and then we would pass "now" to it, then a single "now" can be used consistently across upstream scopes

Reply:
has gone out of scope of this PR as it has been moved to ui-components pkg